net-fs/curlftpfs/curlftpfs-0.9.2.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="Filesystem for accessing FTP hosts"
+DESCRIPTION="A file system for accessing ftp hosts based on FUSE and \
+libcurl.
+As the FTP protocol is not very feature rich, this filesystem does not \
+fulfill every constraint of a real filesystem, but it should be usable \
+for simple tasks like copying and editing files.
+
+CurlFtpFS differentiates itself from other FTP filesystems because it \
+features:
+
+* SSLv3 and TLSv1 support
+* connecting through tunneling HTTP proxies
+* automatically reconnection if the server times out
+* transform absolute symlinks to point back into the ftp file system
+
+Usage on Haiku: First start the userland_fs server:
+	/system/servers/userlandfs_server curlftpfs &
```

Comment:
The intended behaviour is that the kernel automatically invokes the server (if not already running) when you mount (see https://www.haiku-os.org/documents/dev/running_userlandfs). I don't know what mechanism this uses, but it is broken (see https://dev.haiku-os.org/ticket/11750#comment:4 and http://lists.ports.haiku-files.org/pipermail/haikuports-devs-ports.haiku-files.org/2012-June/003271.html). Also, if you want to use any interactive options to your userlandfs plugin (like prompt for a password) you need to pass the options to uselandfs_server instead of when invoking.